dataprofiler/tests/profilers/test_int_column_profile.py

```diff
@@ -277,9 +277,9 @@ def test_profile(self):
                 'bin_edges': np.array([2.0, 10.0/3.0, 14.0/3.0, 6.0])
             },
             quantiles={
-                0: 8.0/3.0,
-                1: 8.0/3.0,
-                2: 4.0
+                0: 8 / 3,  # halfway between 2 and 3.333 (histogram based)
+                1: 10 / 3,  # 50% achieved at 10 / 3 (histogram based)
+                2: 16 / 3,  # halfway between between 14 / 3 and 6
```

Comment:
these are also changing to update with the new histogram calc which uses interpolation